website/templates/emails/mentions.txt.mako

```diff
@@ -0,0 +1,3 @@
+${user} at ${localized_timestamp}: ${content}.
```

Comment:
Oh, didn't notice this was the same as the comment email template; this is probably fine then. Do @mention emails get sent immediately or do they get bundled together over 5-minute time periods (by send_user_email: https://github.com/CenterForOpenScience/osf.io/blob/a5752fa055516ee3dd5fd04613ddaebb723c26b6/website/notifications/tasks.py#L20) ?